### ISSUE TITLE
who has  the Loss Logs in training ?

### ISSUE THREAD

**Issue by ucasiggcas** [xiph/rnnoise#87]
Hi,

In my training, 
the loss in first epoch is about 5,
after about 60 epochs the loss is about 1,
the loss maybe not satisfied,
So who the Loss Logs in training or what loss is the best ?
Should I increase the epoch ?
Or other methods to reduce the loss and val_loss ,who have try ?

Please talk about the question.

Thx

Lychee Tsu

**Comment by ucasiggcas** [comment]
![image](https://user-images.githubusercontent.com/35590066/61795327-bbef8300-ae55-11e9-8afa-949614b50c5e.png)
Not good 
that is so bad
